include/alpaka/math/rsqrt/RsqrtUniformCudaHipBuiltIn.hpp

```diff
@@ -30,22 +30,16 @@ namespace alpaka
         {
             //! The CUDA rsqrt trait specialization.
             template<typename TArg>
-            struct Rsqrt<RsqrtUniformCudaHipBuiltIn, TArg, std::enable_if_t<std::is_arithmetic<TArg>::value>>
+            struct Rsqrt<RsqrtUniformCudaHipBuiltIn, TArg, std::enable_if_t<std::is_arithmetic_v<TArg>>>
```

Comment:
@psychocoderHPC 

The root functions (`cbrt`, `sqrt`, `rsqrt`) are the only floating point operations for which we allow integral parameters, too. Is this intended or an accident?

Reply:
I think `sqrt` should allow integers, as so does the standard library version. For the other two I think makes sense to do same as `sqrt`.